protos/flyteidl/admin/launch_plan.proto

```diff
@@ -93,8 +93,12 @@ message LaunchPlanSpec {
 
     AuthRole auth_role                    = 9;
 
-    // Indicates the runtime priority of the execution. 
+    // Indicates the runtime priority of the execution.
     core.QualityOfService quality_of_service   = 16;
+
+    // Prefix for where offloaded data from user workflows will be written (i.e. Blobs, Schema, query data, etc.)
+    // e.g. s3://bucket/key or s3://bucket/
+    string raw_output_data_prefix            = 17;
```

Comment:
on thinking more, maybe this should be a structure. As we may want to add the policy like
`Sharding desired: True`
`etc`...

Reply:
what else would go in?